src/fcntl.rs

```diff
@@ -64,7 +64,11 @@ libc_bitflags! {
     }
 }
 
-#[cfg(any(feature = "fs", feature = "term"))]
+#[cfg(any(
+    feature = "fs",
+    feature = "term",
+    all(feature = "fanotify", target_os = "linux")
+))]
 libc_bitflags!(
     /// Configuration options for opened files.
     #[cfg_attr(docsrs, doc(cfg(any(feature = "fs", feature = "term"))))]
```

Comment:
This also needs to be updated:
```suggestion
    #[cfg_attr(docsrs, doc(cfg(any(feature = "fs", feature = "term", all(feature = "fanotify", target_os = "linux")))))]
```

Reply:
Oops sorry I missed that one!